utils/iban_calculator.py

```diff
@@ -0,0 +1,153 @@
+import re
+
+
+class IBANCalculator:
```

Comment:
Have you considered using some already existing module for this instead writing your own? e. g. https://pypi.org/project/schwifty/